src/cli.rs

```diff
@@ -432,14 +432,14 @@ pub struct Opt {
     )]
     /// Format string for file hyperlinks (requires --hyperlinks).
```

Comment:
While you are at it, the above (in GitHub I can't add a comment there..) should be `file:///`, not just `//`, see the mentioned RFC 8089.

Reply:
Are you saying that 
```
        default_value = "file://{path}",

```

is incorrect? `{path}` is documented as being an absolute file path, so on Unix will have a leading slash; if we change the default here to 3 slashes then wouldn't we need to make a code change to avoid 4 slashes when concatenating the path? Also we use `"file-line://{path}:{line}"` as an example in the text (two slashes).

Reply:
Right, the path contains the third `/`, then it is fine.

Reply:
Do you mind if we address this in a separate PR? I'm doubtless probably being stupid but it seems to me that if `{path}` is an absolute path, such as `/etc/fstab` then it's correct as-is, yielding `file:///etc/fstab`. I'd prefer not to make a drive-by change that's not related to the PR, without understanding the correctness implications.

Reply:
Yes, it is okay as-is, there is nothing to address :)

See `debug_assert!(absolute_path.as_ref().is_absolute());`